dist/tugboat-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sv_stop() {
+    for s in /etc/service/*; do
+        "$RUNITDIR"/sv stop "$s"
+    done
+}
+
+trap "" SIGCHLD
```

Comment:
If we are trapping the child exit  but not defining a command to run when the signal is triggered, what does this actually accomplish? Curious to know more.

Reply:
Perhaps a comment here is justified!

Reply:
Added a comment to explain.